app/models/saved_claim/veteran_readiness_employment_claim.rb

```diff
@@ -110,6 +110,7 @@ def add_office_location(updated_form)
     updated_form['veteranInformation']&.merge!({ 'regionalOffice' => "#{@office_location} - #{office_name}" })
   end
 
+  # rubocop:disable Metrics/MethodLength
```

Comment:
Platform will reject rubocop disables without justification. We need to break this method up into smaller chunks.

If you aren't comfortable with making that change let me know and we can pull a ruby engineer in.

Reply:
Will work on a change then, thanks for the context!